patches/gcc/0001-Add-Managarm-target.patch

```diff
@@ -105,7 +60,7 @@ index 357b0bed0..570049be8 100644
 +  use_gcc_stdint=wrap
 +
 +  case ${target} in
-+  *-managarm-system*)
++  *-managarm-mlibc*)
```

Comment:
I'm confused. We use `-system` in config.sub but here we replace the existing `-system` patch by `-mlibc`?

Reply:
config.sub upstream has -mlibc, so we should be going for that

Reply:
Agreed

Reply:
But if we do that, we should also use `-mlibc` in config.sub. I'm confused by the mismatch.